sqlup-mode.el

```diff
@@ -199,7 +199,7 @@ ANSI SQL keywords."
       'ansi))
 
 (defun sqlup-redis-mode-p ()
-  (eq major-mode #'redis-mode))
+  (eq major-mode 'redis-mode))
```

Comment:
I'm guessing this is a necessary change, in case you don't have redis-mode installed?